plugins/org.python.pydev/src/org/python/pydev/editor/PyEdit.java

```diff
@@ -669,6 +669,9 @@ private void updateHoverBehavior() {
             String t = types[i];
 
             ISourceViewer sourceViewer = getSourceViewer();
+            if (sourceViewer == null) {
```

Comment:
I think this should be the first thing on this method, not inside this for loop.